tests/test_morphsqueeze.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import pytest
+
+from diffpy.morph.morphs.morphsqueeze import MorphSqueeze
+
+
+class TestMorphSqueeze:
+    @pytest.fixture
+    def setup(self):
```

Comment:
I don't think this needs to be a fixture.  If we are going to reuse it in multiple places we can make it a fixture, but then put it in conftest.py